src/main/java/com/minecolonies/coremod/generation/defaults/DefaultBlockLootTableProvider.java

```diff
@@ -55,6 +64,17 @@ protected void registerTables(@NotNull final LootTableRegistrar registrar)
         saveBlock(ModBlocks.blockColonyWallBanner, registrar);
         saveBlock(ModBlocks.blockIronGate, registrar);
         saveBlock(ModBlocks.blockWoodenGate, registrar);
+        saveBlock(ModBlocks.blockCompostedDirt, lootPool -> {
+            final AlternativesEntry.Builder alternatives = AlternativesEntry.alternatives();
+
+            final LootPoolSingletonContainer.Builder<?> compostedDirt = LootItem.lootTableItem(ModBlocks.blockCompostedDirt);
+            needsSilkTouch(compostedDirt);
+            alternatives.otherwise(compostedDirt);
+
+            alternatives.otherwise(LootItem.lootTableItem(Blocks.DIRT).when(ExplosionCondition.survivesExplosion()));
+
+            lootPool.add(alternatives);
```

Comment:
Builder methods are fluent and should be called as such without great reason to do otherwise.

Also it's nicer to have lambda arguments at the end of the signature so that you don't get that seemingly orphaned `registrar` argument that's easy to overlook.